nautobot_ssot/jobs/base.py

```diff
@@ -51,7 +51,7 @@ class DataSyncBaseJob(BaseJob):  # pylint: disable=too-many-instance-attributes
     dry_run = BooleanVar()
     memory_profiling = BooleanVar(description="Perform a memory profiling analysis.", default=False)
 
-    def load_source_adapter(self):
+    def source_load_adapter(self):
```

Comment:
Not a fan of this function name change myself. Either `load_source_adapter` (my preference) or `source_adapter_load` would be better.